### PR TITLE
Add data-prompt to toolbar.

### PR DESCRIPTION
**Pull request by martinpovolny**
https://bugzilla.redhat.com/show_bug.cgi?id=1388593

review, pls, @himdel? @karelhala?

needs to get to euwe
screenshot_2016-11-04_19-44-19

### CODE REVIEW

**Comment by martinpovolny** [comment]
to test in ManageIQ replace `manageiq-ui-components` in `bower.json` with:

    "manageiq-ui-components": "martinpovolny/ui-components#toolbar_prompt",

File: dist/home/karel/Documents/git/manageiq/ui-components/src/interfaces/toolbar.d.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ export interface IToolbarItem {
 export interface IToolbarData {
     'function': string;
     'function-data': string;
-    targer: string;
+    target: string;
```
Comment (himdel) [inline dist/home/karel/Documents/git/manageiq/ui-components/src/interfaces/toolbar.d.ts]: I *think* we can just delete that whole `dist/home` dir, right @karelhala ?

File: src/toolbar/components/toolbar-menu/toolbar-button.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
         data-click="{{toolbarButton.id}}"
         data-url="{{toolbarButton.url}}"
         data-url_parms="{{toolbarButton.url_parms}}"
+	data-prompt="{{toolbarButton.prompt}}"
```
Comment (himdel) [inline src/toolbar/components/toolbar-menu/toolbar-button.html]: whitespace, also in `toolbar-list` and `toolbar-view`

**Comment by himdel** [comment]
code looks good (except comments :)), but I have not tested..

**Comment by h-kataria** [comment]
@martinpovolny code looks good, but i was not able to verify fix in UI. replacing ``` "manageiq-ui-components": "~0.0.8", ``` with ```"manageiq-ui-components": "martinpovolny/ui-components#toolbar_prompt" ```, in bower.josn did not help.

**Comment by martinpovolny** [comment]
@h-kataria : did you run bower update? you should see in the output that the ui-components gets updated. Then reload all the javascripts and it should work.

**Comment by martinpovolny** [comment]
@himdel : fixed the whitespace, please, check, merge, tag ;-)

**Comment by h-kataria** [comment]
@martinpovolny i ran bin/update, let me try bower update

**Comment by h-kataria** [comment]
@martinpovolny still no prompt after bower update, i noticed this when i ran bower update
```
bower install               manageiq-ui-components#daeb449624

manageiq-ui-components#daeb449624 vendor/assets/bower_components/manageiq-ui-components
```

**Comment by martinpovolny** [comment]
@h-kataria : I forgot, there's a condition in the button code. Here's my diff for a quick test:
```
devil - [~/Projects/manageiq] (master)$ git diff
diff --git a/app/helpers/application_helper/button/collect_logs.rb b/app/helpers/application_helper/button/collect_logs.rb
index 21f6f6e..4402307 100644
--- a/app/helpers/application_helper/button/collect_logs.rb
+++ b/app/helpers/application_helper/button/collect_logs.rb
@@ -1,7 +1,7 @@
 class ApplicationHelper::Button::CollectLogs < ApplicationHelper::Button::Basic
   def calculate_properties
     super
-    if @record.try(:log_file_depot).try(:requires_support_case?)
+    if true # @record.try(:log_file_depot).try(:requires_support_case?)
       self[:prompt] = true
     end
   end
diff --git a/bower.json b/bower.json
index 03ee8f8..1896519 100644
--- a/bower.json
+++ b/bower.json
@@ -38,7 +38,7 @@
     "jquery-ujs": "~1.2.2",
     "jquery.observe_field": "himdel/jquery.observe_field#~0.1.0",
     "kubernetes-topology-graph": "~0.0.23",
-    "manageiq-ui-components": "~0.0.8",
+    "manageiq-ui-components": "martinpovolny/ui-components#toolbar_prompt",
     "moment-strftime": "~0.2.0",
     "moment-timezone": "~0.4.1",
     "numeral": "~1.5.3",
devil - [~/Projects/manageiq] (master)$ 
```

**Comment by h-kataria** [comment]
@martinpovolny unfortunately still no prompt.

**Comment by h-kataria** [comment]
verified in UI :+1:

**Comment by h-kataria** [comment]
@dclarizio please merge, i do not have access to merge PR on this repo.

File: src/toolbar/components/toolbar-menu/toolbarButton.sample.json [hunk 3]
```diff
@@ -16,5 +16,6 @@
   "img_url": "data/some_img.png",
   "explorer": true,
   "url": "reload",
-  "url_parms": "url_parms"
+  "url_parms": "url_parms",
```
Comment (mtho11) [inline src/toolbar/components/toolbar-menu/toolbarButton.sample.json]: FWIW: you can just use the ES6 shorthand properties notation: https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand

Comment (himdel) [inline src/toolbar/components/toolbar-menu/toolbarButton.sample.json]: @mtho11 not in json though ;)

**Pull request merged**